integr8ly/integreatly-operator
# udpate keycloak operator to 9.0.2 (INTLY-6775)

# Description
Update the Keycloak operator to 9.0.2 to pull in CVE
https://issues.redhat.com/browse/INTLY-6775

## How to Verify
- Install RHMI and verify user and cluster sso success
- Run setup-htpass-idp.sh and verify users customer-admin and test-user get added to Keycloak
- Log into a number of products via sso such as 3Scale
- Verify operator versions are updated to 9.0.2
